app/modules/wallet/users/users.route.unit.spec.ts

```diff
@@ -0,0 +1,194 @@
+import * as chai from 'chai';
```

Comment:
Same comments apply here, you can check comments on [this file](https://github.com/thinblock/tb-core-api/pull/11/files#diff-a42219cf082cc412268cedfcde294809)